test/aarch64/test-api-aarch64.cc

```diff
@@ -510,7 +510,7 @@ TEST(isvalid_cpu) {
 }
 
 
-TEST(areconsecutive) {
+TEST(are_consecutive) {
```

Comment:
This appears to be boost or similar and it therefore doesn't care about the symbol name, `_`s are used for others in the file.